src/Rfc6455/Frame.php

```diff
@@ -8,6 +8,7 @@
  * on the root directory of this project
  */
 
+declare(strict_types=1);
```

Comment:
I understand but it is in a central class,  so it should be interesting a load tests for the impact. But ok